api/AudioContext.json

```diff
@@ -90,21 +90,21 @@
           "description": "<code>AudioContext()</code> constructor",
           "support": {
             "chrome": {
-              "version_added": "55",
+              "version_added": "35",
```

Comment:
I tested https://software.hixie.ch/utilities/js/live-dom-viewer/saved/8529 in Chrome 34 and 35, and indeed the unprefixed constructor came in 35. But the prefixed constructor was there earlier, since Chrome 10 if https://caniuse.com/audio-api is correct. Or 14 if the api.AudioContext entry is correct. Can you test old Chrome and update?

BCD is not consistent about whether prefix data should be on the interface, constructor or both. Here, the prefix is repeated for Safari but not for Chrome. I'd prefer for it to be repeated for Chrome as well, since otherwise in the context of https://developer.mozilla.org/en-US/docs/Web/API/AudioContext/AudioContext it will look like the unprefixed constructor is supported going back before Chrome 35, which will be wrong.

Also, this is a good case for adding prefixed constructor tests in mdn-bcd-collector. That there's a prefixed BCD entry for the interface should be enough to infer that the test is needed.